pkgs/firecracker-microvm/firectl/registry.yaml

```diff
@@ -0,0 +1,13 @@
+packages:
+  - type: github_release
+    repo_owner: firecracker-microvm
+    repo_name: firectl
+    description: firectl is a command-line tool to run Firecracker microVMs
+    asset: firectl-{{ .Version }}
+    format: raw
+    supported_envs:
+      - linux
```

Comment:
```suggestion
      - linux/amd64
```

https://github.com/firecracker-microvm/firectl/releases

I don't think this binary works on Linux ARM64.

Reply:
Fixed. 7d279858d14102c3d4660b64d972f9e8aab05c7e